src-arduino/examples/pn532_read/pn532_read.ino

```diff
@@ -0,0 +1,136 @@
+#include <SoftwareSerial.h>
+#include <PN532_SWHSU.h>
+#include <PN532.h>
+
+// This doesnt work for esp32s2 LOlE since there is no build for it.
+SoftwareSerial SWSerial(12, 11); // RX, TX
+PN532_SWHSU pn532swhsu(SWSerial);
+PN532 nfc(pn532swhsu);
+
+void setup(void)
+{
+
+  Serial.begin(115200);
```

Comment:
tf is 115200 O_O

Reply:
this is a common serial bitrate, very understood magic number